libs/payments/currency/src/lib/currency.manager.spec.ts

```diff
@@ -13,14 +13,14 @@ import {
 import { CURRENCIES_TO_COUNTRIES } from './currency.constants';
 
 describe('CurrencyManager', () => {
-  let mockCurrencyManager: CurrencyManager;
+  let currencyManager: CurrencyManager;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
       providers: [CurrencyManager],
     }).compile();
 
-    mockCurrencyManager = module.get<CurrencyManager>(CurrencyManager);
+    currencyManager = module.get<CurrencyManager>(CurrencyManager);
```

Comment:
This swap makes sense -- you're getting a real currencyManager here, not a mock.